models/build.js

```diff
@@ -106,6 +106,7 @@ const MODEL = {
 
     status: Joi
         .string().valid([
+            'CREATING_CONTAINER',
```

Comment:
Who is going to update the build status to `CREATING_CONTAINER `? The API set it to `QUEUED` and the build itself update it to `Running`. Who's going to be the middle man?